eth/backend.go

```diff
@@ -84,17 +82,33 @@ func (b *backend) SendTransaction(ctx context.Context, tx *types.Transaction) er
 	b.nonceManager.Unlock(sender)
 
 	data := tx.Data()
-	method, ok := b.methods[string(data[:4])]
-	if !ok {
-		method = "unknown"
+	methodSig := string(data[:4])
+	method, err := b.abiMap[methodSig].MethodById(data[:4])
```

Comment:
If we set `methodSig = data[:4]` then we can reuse it so we only have to index into `data` once (minimizes the chance of accidentally passing in the wrong index value). Then, we can pass `string(methodSig)` to `decodeTxParams()`.

Reply:
fixed in b47e031